gallery/src/components/NxSmallTag/NxSmallTagPage.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxP, NxTable, NxTextLink } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxSmallTagExample from './NxSmallTagExample';
+import NxSmallTagInTableExample from './NxSmallTagInTableExample';
+
+const NxSmallTagExampleCode = require('./NxSmallTagExample?raw');
+const NxSmallTagInTableExampleCode = require('./NxSmallTagInTableExample?raw');
+
+const NxSmallTagPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Small tags are used only by designers and developers when making a product, rather than
+        being applied and removed by users of the product. <NxCode>NxSmallTag</NxCode> is an inline component
+        intended to call out product features ("Beta", "Custom", "Labs", etc), and is informational
+        whereas <NxCode>NxTag</NxCode> is functional. There is a set of colors for small tags that can be specified
+        via props.
+      </NxP>
```

Comment:
Might also want to mention that small tags have their own set of colors and specifically do not use the `SelectableColors`.

Reply:
Updated here 16bdea34c720239d28187b4fa8eb807a5cf20b5e.